service/k8sclient/k8sclient.go

```diff
@@ -506,6 +512,24 @@ func (c *K8sClient) CreatePXCCluster(ctx context.Context, params *PXCParams) err
 			},
 		},
 	}
+	bytes, err := ioutil.ReadFile(pxcCRFile)
+	// Ignore error if we failed to open cr file
+	if err == nil {
```

Comment:
probably we need a little bit more logging here. To log that we actually have overrides, and maybe as debug - what overrides (if that doesn't contain some secrets).